_data/locales/en.yaml

```diff
@@ -0,0 +1,25 @@
+site_title: openSUSE Documentation
+site_title_short: Documentation
+site_description: High quality openSUSE documentation for both sysadmins and desktop users.
```

Comment:
I'd be happier without the "high quality" thing being thrown in there. (I am obviously not a marketing person.)